Application/Telegram/Commands/FinishBookingCommand.cs

```diff
@@ -30,9 +30,12 @@ public async Task Execute(CallbackQuery callbackQuery)
                         WorkplaceId = currentUserInfo.WorkplaceId,
                         StartDate = Helper.GetStartDate(callbackQuery),
                         EndDate = Helper.GetEndDate(callbackQuery),
-                        IsRecurring = false,
+                        IsRecurring = currentUserInfo.RecurringDay is null? false : true,
```

Comment:
It could be !(currentUserInfo.RecurringDay is null)

Reply:
done